packages/modules-runtime/server.js

```diff
@@ -62,20 +62,12 @@ Module.prototype.useNode = function () {
   if (parts.indexOf("node_modules", start) < 0) {
     // Don't try to use Node for modules that aren't in node_modules
     // directories.
-    return false;
-  }
-
-  try {
-    npmRequire.resolve(this.id);
-  } catch (e) {
-    return false;
+    throw new Error('useNode can not be used to import modules outside of node_modules');
```

Comment:
We might be able to remove this logic since the import scanner already checked it before adding the call to `useNode`.